### ISSUE TITLE
Wiki does not have an article on mono audio

### ISSUE THREAD

**Issue by Zemogiter** [astronautlevel2/Anemone3DS#327]
If one were to install a custom theme with mono audio (which is known to cause issues in home menu), a warning is displayed to check the wiki.
Thing is, there is nothing on mono audio on this repo's wiki.

**Comment by astronautlevel2** [comment]
Good point! I meant to write something up before the release but forgot to. There's not that much to say, since the Home Menu just always expects dual channel and doesn't properly clear the buffers for mono audio, leading to weird artifacts and heavily messed up sound.

**Comment by Zemogiter** [comment]
We could have an article saying how to tell from a glance which themes have mono audio, or better yet - add an functionality to the theme manager that displays a icon/warning when hovering over such themes.

**Comment by astronautlevel2** [comment]
I dislike the latter idea because it would add a decent bit of overhead having to read BGM files while navigating the theme list, I think warning the user when they install a mono audio BGM theme is enough for the app. I'll put something up on the wiki describing the issues & next steps.

**Comment by Zemogiter** [comment]
Understood

**Comment by signer-ink-beast** [comment]
Just ran into this and went looking for the non-existent article. Reading some GBA Temp threads gave me the gist of the issues.

Would it be possible to tell the user which themes have mono audio when installed in shuffle mode (checking 10 at most, once the user starts a shuffle install) either before or during installation? Maybe even a quick toggle to install those theme(s) with mono audio without music?

Right now one has to do a lot of finagling about. Apparently one can only tell there's an issue easily with headphones.

**Comment by Sylverena** [comment]
This may be out of the scope of this issue however I do want to note that I have tried to install a theme and got a warning about mono audio - however I didn't notice any audio artifacting, stuttering, nothing. Everything seemed to work as expected.

**Comment by onyxcode** [comment]
What are said issues with using mono audio? I can't find info on it

**Comment by NicholasC2** [comment]
"The Home Menu just always expects dual channel and doesn't properly clear the buffers for mono audio, leading to weird artifacts and heavily messed up sound." - response from [astronautlevel2](https://github.com/astronautlevel2)
